src/test/ui/deprecation/atomic_initializers.stderr

```diff
@@ -2,11 +2,7 @@ warning: use of deprecated item 'std::sync::atomic::ATOMIC_ISIZE_INIT': the `new
   --> $DIR/atomic_initializers.rs:8:27
    |
 LL | static FOO: AtomicIsize = ATOMIC_ISIZE_INIT;
-   |                           ^^^^^^^^^^^^^^^^^
+   |                           ^^^^^^^^^^^^^^^^^ help: replace the usage of the deprecated item: `AtomicIsize::new(0)`
    |
    = note: #[warn(deprecated)] on by default
-help: use of deprecated item 'std::sync::atomic::ATOMIC_ISIZE_INIT': the `new` function is now preferred
```

Comment:
Why has this been removed? The `reason` in `rustc_deprecated` now seems to be ignored.

Reply:
It hasn't been removed. The reason is already part of the error message.

Reply:
Oh, I see: I didn't spot that a line was missing in GitHub's diff view.